draft-ietf-taps-interface.md

```diff
@@ -3237,3 +3237,9 @@ applications can obtain this information via the `Sent` Event ({{sent}}).
 
 * Notification to a receiver that a partial message delivery has been aborted:
 `ReceiveError` Event ({{receive-error}}).
+
+
+## Unsupported Elements of the Minimal Set
+
+* Notification of Excessive Retransmissions (early warning below abortion threshold):
+This is not supported because it is TCP-specific and hardly
```

Comment:
This is incomplete?

Reply:
Argh!  More importantly, it doesn't belong here - this is an accident  :(   will fix